sale_product_set/models/product_set.py

```diff
@@ -8,4 +8,13 @@ class ProductSet(models.Model):
 
     name = fields.Char(_('Name'), help=_('Product set name'), required=True)
     set_line_ids = fields.One2many(
-        'product.set.line', 'product_set_id', _(u"Products"))
+        'product.set.line', 'product_set_id', _(u"Products"), copy=True)
```

Comment:
@pedrobaeza, 
cc @moylop260 

I don't really know how that's works but I guess this kind of issue could be track by maintainer-quality-tools lint tools or [moylop260/pylint-oca](github.com/moylop260/pylint-oca) project ?

Is there a list of implemented (or expected) lint issues?